shared/src/main/scala/com/cibo/evilplot/geometry/Drawable.scala

```diff
@@ -87,12 +87,18 @@ final case class Path(points: Seq[Point], strokeWidth: Double) extends Drawable
   private lazy val xS: Seq[Double] = points.map(_.x)
   private lazy val yS: Seq[Double] = points.map(_.y)
   lazy val extent: Extent =
-    if (points.nonEmpty) Extent(xS.max - xS.min, yS.max - yS.min) else Extent(0, 0)
+    if (points.nonEmpty) Extent(xS.max , yS.max) else Extent(0, 0)
   def draw(context: RenderContext): Unit = if (points.nonEmpty) context.draw(this) else ()
 }
 object Path {
-  implicit val encoder: Encoder[Path] = deriveEncoder[Path]
-  implicit val decoder: Decoder[Path] = deriveDecoder[Path]
+
+  def apply(points: Seq[Point], strokeWidth: Double) : Drawable = {
+    val minX = points.map(_.x).min
+    val minY = points.map(_.y).min
+    Translate(new Path(points.map{x=> Point(x.x - minX, x.y - minY)},strokeWidth), minX, minY)
+  }
+  implicit val encoder: Encoder[Path] = Encoder.forProduct2("points", "strokeWidth"){x => (x.points, x.strokeWidth)}
```

Comment:
These encoders were previously using a circe configuration to shorten the field names (see `com.cibo.evilplot.JSONUtils.minifyProperties`). "p" and "s" as names in the JSON would keep them the same as they are now.